src/main/java/com/ekart/thunderbolt/entity/Product.java

```diff
@@ -0,0 +1,55 @@
+package com.ekart.thunderbolt.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "product")
+@Builder
```

Comment:
@Builder uses builder pattern, which a little different from the @Data which is a combination of getter and setter. If you're using builder, you need to only have @Setter instead of @Data.